sources/packages/api/src-sql/SFASIndividuals/Create-sfas-individuals.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE IF NOT EXISTS sims.sfas_individuals (
+  id INT PRIMARY KEY,
+  first_name VARCHAR(50) NOT NULL,
```

Comment:
nullable, due to mononymous names

Reply:
Changed to accept null.
It will be null when the first name and middle name are both nulls.